arangod/RocksDBEngine/RocksDBVPackIndex.cpp

```diff
@@ -420,9 +531,10 @@ class RocksDBVPackIndexIterator final : public IndexIterator {
   arangodb::RocksDBVPackIndex const* _index;
   RocksDBVPackComparator const* _cmp;
   std::unique_ptr<rocksdb::Iterator> _iterator;
-  RocksDBKeyBounds const _bounds;
+  RocksDBKeyBounds _bounds;
   // used for iterate_upper_bound iterate_lower_bound
   rocksdb::Slice _rangeBound;
+  RocksDBVPackIndexSearchValueFormat _format;
```

Comment:
This can probably be const.
```suggestion
  RocksDBVPackIndexSearchValueFormat const _format;
```